services/emailer/src/namex_emailer/email_processors/nr_notification.py

```diff
@@ -105,6 +111,7 @@ def process(email_info: SimpleCloudEvent, option) -> dict:  # pylint: disable-ms
     corp_online_url = current_app.config.get("COLIN_URL")
     form_page_url = current_app.config.get("CORP_FORMS_URL")
     societies_url = current_app.config.get("SOCIETIES_URL")
+    magic_link = f'{current_app.config.get("AUTH_WEB_URL")}magicLink/?nrId={nr_number}&email={nr_data["applicants"]["emailAddress"]}&phone={nr_data["applicants"]["phoneNumber"]}'
```

Comment:
Generate the magic link

Reply:
Kevin, I'm changing the frontend a bit. The link that the FE expects now is of the form:
[/incorporateNow?nr=NR+0521856&phone=1112223333](https://business-registry-dev--pr-37-8gkpatut.web.app/incorporateNow?nr=NR+0521856&phone=1112223333)